docs/self-managed/setup/deploy/amazon/aws-ec2.md

```diff
@@ -0,0 +1,352 @@
+---
+id: aws-ec2
+title: "Amazon EC2"
+description: "Learn how to install Camunda 8 on AWS EC2 instances."
+---
+
+This guide provides a detailed walkthrough for installing the Camunda 8 single JAR on AWS EC2 instances. It focuses on managed services by AWS and their cloud offering. Finally, you will verify that the connection to your Self-Managed Camunda 8 environment is working.
+
+This guide focuses on setting up the [orchestration cluster](/self-managed/reference-architecture/reference-architecture.md#orchestration-cluster-vs-web-modeler-and-console) for Camunda 8. The Web Modeler and Console are not covered in this manual deployment approach. These components are supported on Kubernetes and should be [deployed using Kubernetes](/self-managed/setup/install.md/#install-web-modeler).
+
+## Disclaimer
```

Comment:
I think we can eliminate this header, since it's just for the note, and doesn't necessarily need the legal weight of a disclaimer? 

If the header feels necessary, I would remove the content from a note, and retitle the section to "Use other cloud providers".